### PR TITLE
Download `dof_sales` table directly from Open Data

### PR DESCRIPTION
**Pull request by sraby**
This PR changes the source of the `dof_sales` table—originally we downloaded 5 Excel spreadsheets from the DOF website, and now we can download the complete table directly from NYC Open Data. 

This PR also includes some updates to the `test_nycdb.py` file, which due to the `psychopg2` package updating was raising [a new error](https://github.com/psycopg/psycopg2/issues/1316).

### CODE REVIEW

File: src/nycdb/dof_parser.py [hunk 1]
```diff
@@ -1,61 +0,0 @@
-"""
-DOF rolling sales data is downloadable only as an excel document.
-This file process that file so that we can treat like we do csvs.
-"""
-import xlrd
```
Comment (toolness) [inline src/nycdb/dof_parser.py:5]: Are there any modules left that use `xlrd`?  If this was the last one, we might be able to remove it as a dependency from nycdb!

**Review by toolness — APPROVED** [review]
This looks great, just one comment!

File: src/nycdb/dof_parser.py (hunk 1 above)
Comment (toolness) [reply src/nycdb/dof_parser.py:5]: If it is the last place (and it looks like it is) you should remove this line too:

https://github.com/nycdb/nycdb/blob/6dc3442b729b49042ee11bb31fe997f0fbb160be/src/setup.py#L37

**Comment by sraby** [comment]
@lblok check it out!

**Pull request merged**